reference/hoon-expressions/rune/sig.md

```diff
@@ -314,24 +314,28 @@ From the kernel:
 ### ~& "sigpam"
 
 `[%sgpm p=hoon q=hoon]`: debugging printf.
+`[%sgpm p=(stun [1 3] gar) q=hoon r=hoon]`: debugging printf with priority.
 
 ##### Expands to
 
 `q`.
 
 ##### Product
 
-Pretty-prints `p` on the console before computing `q`.
+Pretty-prints `p` on the console before computing `q`. If a priority is given, then prettyprint `p` with priority equal to the number of gar characters.
 
 ##### Discussion
 
 This rune has no semantic effect beyond the Hoon expression `q`.  It's used solely to create a side-effect: printing the value of `p` to the console.
 
+If the priority is nonzero then the text will be colored green, yellow or red in increasing order.
```

Comment:
It'd be better to say something like: "The priority can be used to filter the hints, or accentuate them when printing. Currently, hints with a nonzero priority are colored green, yellow or red in increasing order."

There's nothing about those colors intrinsic to the priority levels. They were a recent change, and might easily change again in the future. The priorities are basically just like log levels.